src/services/billing/cost-management/cost-analysis/modules/CostAnalysisDataTable.vue

```diff
@@ -219,19 +227,24 @@ export default {
                 },
             };
         };
+        const getIsRaised = (item, fieldName) => {
+            const currDate = fieldName.split('.')[1];
+            const prevDate = dayjs.utc(currDate).subtract(1, state.timeUnit).format(state.dateFormat);
+            const currValue = item.usd_cost[currDate];
+            const prevValue = item.usd_cost[prevDate];
+            if (prevValue) return ((currValue - prevValue) / prevValue) * 100 > 50;
```

Comment:
```suggestion
            const currValue = item.usd_cost[currDate] ?? 0;
            const prevValue = item.usd_cost[prevDate] ?? 0;
            if (prevValue) return ((currValue - prevValue) / prevValue) * 100 > 50;
```

undefined 인 경우에 대한 가딩이 필요해보입니다.

Reply:
옙!! 근데 prevValue의 경우 undefined면 걍 넘어가는데 그래도 필요한가유?